apps/web/app/api/ably-auth/route.ts

```diff
@@ -0,0 +1,34 @@
+import { isProd } from "@/constants/contracts";
+import { CHANGE_EVENT_CHANNEL_NAME } from "@/globals";
+import Ably from "ably";
+import { NextResponse, NextRequest } from "next/server";
+
+export async function POST(req: NextRequest, res: NextResponse) {
+  // Used for linter that fails
+  if (!process.env.NEXT_ABLY_API_KEY) {
```

Comment:
put the env in the sample/example env file

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/5e63b772ae467e771b9f8d8b1cb1b915a8838fd8